admin/templates/Includes/LeftAndMain_Menu.ss

```diff
@@ -1,36 +1,10 @@
 <div class="cms-menu cms-panel cms-panel-layout west" id="cms-menu" data-layout-type="border">
 	<div class="cms-logo-header north">
```

Comment:
https://github.com/silverstripe/silverstripe-subsites/blob/master/templates/LeftAndMain_Menu.ss is an example of one override.